pdns/dnsdist-lua-rules.cc

```diff
@@ -479,6 +479,10 @@ void setupLuaRules()
       return std::dynamic_pointer_cast<DNSRule>(tisr);
     });
 
+  g_lua.registerFunction<void(std::shared_ptr<TimedIPSetRule>::*)()>("__toString", [](std::shared_ptr<TimedIPSetRule> tisr) {
```

Comment:
shouldn't this also be `__tostring` (lowercase) then?

Reply:
Thanks.